index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"
+        <title></title>
+        <link rel="stylesheet" href="./index.css">
+    </head>
+    <body>
+<div class="card">
+    <div class="search">
+        <input type="text" placeholder="Enter preferred city" spellcheck="false">
+        <button>
+            <img src="./images/search.png">
+        </button>
+    </div>
+    <div class="error">
+        <p> Invalid city name</p>
+    </div>
+    <div class="weather">
+        <img src="./images/rain.png" class="weather-icon">
+        <h1 class="temperature">22°c</h1>
+        <h2 class="city">Lagos</h2>
+        <div class="details">
+            <div class="first-part">
+                <img src="./images/wind.png">
+            <div>
+                <p class="windy">20km/h</p>
+                <p>Wind Speed</p>
+            </div>
+            </div>
+            <div class="first-part">
+                <img src="./images/humidity.png">
+            <div>
+                <p class="humidity">50%</p>
+                <p>Humidity</p>
+            </div>
+            </div>
+        </div>
+
+    </div>
+</div>
+
+<script>
+const apiKey = "e39e501fcd51e7187f1a879817e82803";
+const apiUrl = "https://api.openweathermap.org/data/2.5/weather?&unit=metric&q=";
+
+const searchInput = document.querySelector('.search input');
+const searchBtn = document.querySelector('.search button');
+const weatherIcon = document.querySelector('.weather-icon');
+
+async function getTheWeather(city) {
+    const response = await fetch(apiUrl + city + `&appid=${apiKey}`);
+    const data = await response.json();
+if (response.status == 404) {
+    document.querySelector('.error').style.display = 'block';
+    document.querySelector('.weather').style.display = 'none';
+} else {
+
+document.querySelector('.city').innerHTML = data.name;
+document.querySelector('.temperature').innerHTML = Math.round(data.main.temp) +'°c';
+document.querySelector('.windy').innerHTML = data.wind.speed + 'km/h';
+document.querySelector('.humidity').innerHTML = data.main.humidity + '%';
+
+   
+if (data.weather[0].main == 'Clear') {
+    weatherIcon.src ="./images/clear.png";
+} 
+else if(data.weather[0].main == 'Clouds') {
+    weatherIcon.src ="./images/clouds.png";
+}
+else if(data.weather[0].main == 'Rain') {
+    weatherIcon.src = "./images/rain.png";
+}
+else if(data.weather[0].main == 'Drizzle') {
+    weatherIcon.src = "./images/drizzle.png";
+}
+else if(data.weather[0].main == 'snow') {
+    weatherIcon.src = "./images/snow.png";
+}
+else if(data.weather[0].main == 'Mist') {
+    weatherIcon.src ="./images/mist.png"
```

Comment:
Please convert this 

```
f (data.weather[0].main == 'Clear') {
    weatherIcon.src ="./images/clear.png";
} 
else if(data.weather[0].main == 'Clouds') {
    weatherIcon.src ="./images/clouds.png";
}
else if(data.weather[0].main == 'Rain') {
    weatherIcon.src = "./images/rain.png";
}
else if(data.weather[0].main == 'Drizzle') {
    weatherIcon.src = "./images/drizzle.png";
}
else if(data.weather[0].main == 'snow') {
    weatherIcon.src = "./images/snow.png";
}
else if(data.weather[0].main == 'Mist') {
    weatherIcon.src ="./images/mist.png"
}

```

to a switch statement and it should come from a helper file, say helper.util.js